install_config/router/f5_router.adoc

```diff
@@ -243,27 +241,27 @@ BIG-IP® documentation] for more information.
 . xref:f5-configuring-the-virtual-servers[Configure a virtual server] for the
 custom partition.
 
-. Deploy the F5 router using the `--external-host-partition-path` flag to specify a partition path:
+. Deploy the F5 router plug-in using the `--external-host-partition-path` flag to specify a partition path:
 +
 ----
 $ oc adm router --external-host-partition-path=/OpenShift/zone1 ...
 ----
 
 
 [[setting-up-f5-native-integration-with-openshift]]
-== Setting Up F5 Native Integration
+== Setting Up F5 Router Plug-in
 
 [NOTE]
 ====
 This section reviews how to set up F5 native integration with {product-title}.
-The concepts of F5 appliance and {product-title} connection and data flow of F5
-native integration are discussed in the
-xref:../../architecture/networking/assembly_available_router_plugins.adoc#architecture-f5-native-integration[F5 Native Integration] section of the Routes topic.
+The concepts of the F5 appliance and {product-title} connection and data flow of the F5
+router plug-in are discussed in the
+xref:../../architecture/networking/assembly_available_router_plugins.adoc#architecture-f5-native-integration[F5 Router Plug-in] section of the Routes topic.
 ====
 
 [NOTE]
 ====
-Only *F5 BIG-IP®* appliance version 12.x and above works with the native integration
+Only *F5 BIG-IP* appliance version 12.x and above works with the F5 router plug-in
```

Comment:
This line seems to contradict line 69. Will you confirm?

Reply:
@rajatchopra I think you were the original author of this doc. in one place it says 11.x and 12.x, but here it says 12.x and above (which would include 12.x, 13,x and now 14.x which recently released--and I'm fairly certain your teams have not tested F5 router plug-in with 14.x) 
How should we re-write these sections to be consistent? I suggest 
Only *F5 BIG-IP* appliance versions 11.x and 12.x works with the F5 router plug-in

thoughts?